QUICK_START_GUIDE.md

```diff
@@ -0,0 +1,272 @@
+# Quick Start Guide
+
+## Prerequisites
+* Ubuntu 18.04 / 20.04
+* Python 3.8+
+* [OpenVINO™](https://software.intel.com/en-us/openvino-toolkit) - for exporting and running models
```

Comment:
I think that it is possible to remove OpenVINO from the requirements, because packages `openvino` and `openvino-dev` will be installed using pip.
Plus, when installing on Ubuntu 18.04 using the installer (by link), there may be problems with the Python version for IE.

Reply:
removed